src/components/NavigationBar.js

```diff
@@ -20,13 +24,13 @@ class NavigationBar extends Component {
 
   render() {
     return (
-      <Navbar color="#000" dark expand="md">
+      <Navbar color={NAV_BAR_COLOUR} dark expand="md">
```

Comment:
This prop doesn't actually do anything so you could probably just remove it and the constant. 
I'm setting the background via css 🙈